bot/core.py

```diff
@@ -19,23 +18,21 @@ def __init__(self, bot) -> None:
         self.bot = bot
 
     async def chat_and_save(self, local_chain: ConversationCache, input: str) -> tuple[str, str]:
-        thought_chain =  OBJECTIVE_THOUGHT_CHAIN 
-        response_chain = OBJECTIVE_RESPONSE_CHAIN # if local_chain.conversation_type == "discuss" else WORKSHOP_RESPONSE_CHAIN
+        bloom_chain =  BLOOM_CHAIN # if local_chain.conversation_type == "discuss" else WORKSHOP_RESPONSE_CHAIN
         # response_chain = local_chain.conversation_type == "discuss" ? DISCUSS_RESPONSE_CHAIN : WORKSHOP_RESPONSE_CHAIN
 
         thought = await chat(
             inp=input,
-            thought_chain=thought_chain,
+            thought_chain=bloom_chain,
             thought_memory=local_chain.thought_memory
         )
         response = await chat(
             inp=input,
             thought=thought,
-            response_chain=response_chain,
+            response_chain=bloom_chain,
             response_memory=local_chain.response_memory
         )
```

Comment:
So if we changed the interface in bloom chain to have `think` and `respond` methods is there a reason to have two different calls to chat. I would recommend we can refactor chat to automatically return the thought and response under the hood. 

I'm guessing we did this for backwards compatibility sake, but at this stage not a huge concern. Rather the code be clean.